src/test/kotlin/lotto/CalculatorTest.kt

```diff
@@ -0,0 +1,34 @@
+package lotto
+
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class CalculatorTest {
+    @Test
+    fun `Count number of matches between Lotto ticket and winning numbers`() {
+        val lotto = Lotto(listOf(3, 8, 27, 30, 35, 44))
+        val numbers = Lotto(listOf(3, 8, 27, 4, 5, 6))
+        val winningNumbers = WinningNumbers(numbers, 7)
+
+        val calculator = Calculator(mutableListOf<Lotto>(lotto), winningNumbers)
+        assertEquals(calculator.results.contains(Rank.FIFTH), true)
+    }
```

Comment:
The three terms `lotto`, `numbers`, and `lottoTicket` are being used inconsistently.
How about unifying them into one consistent term?
This way, fellow developers won't get confused!